src/main/java/fi/otavanopisto/kuntaapi/server/integrations/management/ManagementFragmentEntityUpdater.java

```diff
@@ -0,0 +1,175 @@
+
+  private static final int TIMER_INTERVAL = 1000;
+
+  @Inject
+  private Logger logger;
+  
+  @Inject
+  private ManagementTranslator managementTranslator;
+  
+  @Inject
+  private ManagementApi managementApi;
+  
+  @Inject
+  private IdentifierController identifierController;
+
+  @Inject
+  private FragmentCache fragmentCache;
+  
+  @Inject
+  private ModificationHashCache modificationHashCache;
+  
+  @Resource
+  private TimerService timerService;
+
+  private boolean stopped;
+  private IdUpdateRequestQueue<FragmentIdUpdateRequest> queue;
+
+  @PostConstruct
+  public void init() {
+    queue = new IdUpdateRequestQueue<>(ManagementConsts.IDENTIFIER_NAME);
+  }
+
+  @Override
+  public String getName() {
+    return "management-fragments";
+  }
+
+  @Override
+  public void startTimer() {
+    startTimer(TIMER_INTERVAL);
+  }
+
+  private void startTimer(int duration) {
+    stopped = false;
+    TimerConfig timerConfig = new TimerConfig();
+    timerConfig.setPersistent(false);
+    timerService.createSingleActionTimer(duration, timerConfig);
+  }
+
+  @Override
+  public void stopTimer() {
+    stopped = true;
+  }
+  
+  @Asynchronous
+  public void onFragmentIdUpdateRequest(@Observes FragmentIdUpdateRequest event) {
+    if (!stopped) {
+      FragmentId fragmentId = event.getId();
+      
+      if (!StringUtils.equals(fragmentId.getSource(), ManagementConsts.IDENTIFIER_NAME)) {
+        return;
+      }
+      
+      queue.add(event);
+    }
+  }
+  
+  @Asynchronous
+  public void onFragmentIdRemoveRequest(@Observes FragmentIdRemoveRequest event) {
+    if (!stopped) {
+      FragmentId fragmentId = event.getId();
+      
+      if (!StringUtils.equals(fragmentId.getSource(), ManagementConsts.IDENTIFIER_NAME)) {
+        return;
+      }
+      
+      deleteFragment(event, fragmentId);
+    }
+  }
+  
+  @Timeout
+  public void timeout(Timer timer) {
+    if (!stopped) {
+      FragmentIdUpdateRequest updateRequest = queue.next();
+      if (updateRequest != null) {
+        updateManagementFragment(updateRequest.getOrganizationId(), updateRequest.getId(), updateRequest.getOrderIndex());
+      }
+
+      startTimer(SystemUtils.inTestMode() ? 1000 : TIMER_INTERVAL);
+    }
+  }
+  
+  private void updateManagementFragment(OrganizationId organizationId, FragmentId fragmentId, Long orderIndex) {
+    DefaultApi api = managementApi.getApi(organizationId);
+    
+    ApiResponse<Fragment> response = api.wpV2FragmentIdGet(fragmentId.getId(), null, null);
+    if (response.isOk()) {
+      updateManagementFragment(organizationId, response.getResponse(), orderIndex);
+    } else {
+      logger.warning(String.format("Find organization %s fragment %s failed on [%d] %s", organizationId.getId(), fragmentId.toString(), response.getStatus(), response.getMessage()));
+    }
+  }
+  
+  private void updateManagementFragment(OrganizationId organizationId, Fragment managementFragment, Long orderIndex) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "updateManagementFragment" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)